lib/fast_gettext/mo_file.rb

```diff
@@ -5,6 +5,8 @@ module FastGettext
   class MoFile
     PLURAL_SEPERATOR = "\000"
 
+    attr_reader :data
```

Comment:
I made it public to enable `Merge` accessing the data directly and merge it into it's own structure. I tried to explain it more in a comment below.